ui/translations/en-us.yaml

```diff
@@ -148,3 +148,32 @@ status_report_indicator:
     checking_now: Checking now…
     last_checked: Last checked
     unknown: Status unknown
+
+app_card_build:
```

Comment:
[sand] Is there a reason why we need to have the same text separated for the `app_card_build` and `app_item_build` (besides the additional `built_in` text)? Could we make it more generic like `app_build`?

Reply:
We absolutely could. My thought here was that we might want to leave ourselves the option of having much terser text in `<AppCard::Build>` because there’s so much less space to play with.

Reply:
Thought about it some more and decided you were right. Applied your suggestion in f0aa30c08ce4caac23402f068cf00320b9530d1e